framework/infrastructure/k8s.py

```diff
@@ -75,6 +77,25 @@
 )
 
 
+class RouteKinds(enum.Enum):
```

Comment:
Nit: Class name should be singular, i.e. `RouteKind`

Reply:
Good point, thanks!